ui/src/SmsTemplatesList.js

```diff
@@ -154,13 +156,17 @@ class SmsTemplatesList extends Component {
     } else {
       try {
         const parsedJson = JSON.parse(this.state.template);
-        if (!Array.isArray(parsedJson)) {
+        const hasDuplicateTemplateColumns = new Set(parsedJson).size !== parsedJson.length;
+        if (!Array.isArray(parsedJson) || hasDuplicateTemplateColumns) {
           this.setState({ templateValidationError: true });
           failedValidation = true;
+          this.setState({ templateValidationErrorMessage: "sms template must be JSON array with one or more unique elements" });
```

Comment:
Not seeing this validation message. I'm seeing `Error Creating SMSTemplate` instead